cmd/protoc-gen-go-grpc/grpc.go

```diff
@@ -187,6 +187,17 @@ func generateFileContent(gen *protogen.Plugin, file *protogen.File, g *protogen.
 	}
 }
 
+// Copy comments from proto file to _grpc.pb.go file.
+func copyCommentsFromProto(g *protogen.GeneratedFile, service *protogen.Service) {
+
+	// Pint comment without '\n', if comment has '\n', godoc considers this annotation to be non-service annotation.
+	if service.Comments.Leading != "" {
+		// Add empty comments line for godoc.
```

Comment:
How about:

`// Add empty comment line to attach this service's comments to the godoc comments previously output for all services.`

?